packages/web/src/session/session.ts

```diff
@@ -89,15 +110,29 @@ function hasNativeSessionId(): boolean {
 	return typeof window !== 'undefined' && window['SplunkRumNative'] && window['SplunkRumNative'].getNativeSessionId
 }
 
-class ActivitySpanProcessor implements SpanProcessor {
+class SessionSpanProcessor implements SpanProcessor {
+	constructor(
+		private readonly allSpansAreActivity: boolean,
+		private readonly useLocalStorage: boolean,
```

Comment:
This might be an object, which prevents misordering of arguments.

Reply:
https://github.com/signalfx/splunk-otel-js-web/pull/899/commits/dbd46636eeba1353e8848500ad19e187a323529c